src/ui/triggers_main_area.ui

```diff
@@ -1162,7 +1162,7 @@
                </size>
               </property>
               <property name="toolTip">
-               <string>Only the filtered content (=capture groups) will be passed on to child triggers, not the initial line (see manual on filters)</string>
+               <string/>
```

Comment:
The Qt Designer plugin/stand alone utility will not do it after a tool-tip has previously been in place but using the plain text editor you may safely remove this (now empty) line and the surrounding `<property ...>`...`</property>` XML tag-pair.